src/components/Dialog/SettingDialog/FileNameTemplateDialog.stories.ts

```diff
@@ -0,0 +1,144 @@
+import { userEvent, within, expect } from "@storybook/test";
+
+import { Meta, StoryObj } from "@storybook/vue3";
+import FileNameTemplateDialog from "./FileNameTemplateDialog.vue";
+import {
+  buildAudioFileNameFromRawData,
+  DEFAULT_AUDIO_FILE_BASE_NAME_TEMPLATE,
+} from "@/store/utility";
+
+const meta: Meta<typeof FileNameTemplateDialog> = {
+  component: FileNameTemplateDialog,
+  args: {
+    availableTags: [
+      "index",
+      "characterName",
+      "styleName",
+      "text",
+      "date",
+      "projectName",
+    ],
+    defaultTemplate: DEFAULT_AUDIO_FILE_BASE_NAME_TEMPLATE,
+    savedTemplate: "",
+    fileNameBuilder: buildAudioFileNameFromRawData,
+  },
+  tags: ["!autodocs"], // ダイアログ系はautodocsのプレビューが正しく表示されないので無効化
+};
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Opened: Story = {
+  name: "開いている",
+  args: {
+    openDialog: true,
+  },
+};
+
+/** ファイル名パターンをクリアして入力する */
+const clearAndInput = async (inputValue: string) => {
+  const canvas = within(document.body); // ダイアログなので例外的にdocument.bodyを使う
+  const input = canvas.getByLabelText<HTMLInputElement>("ファイル名パターン");
+  await userEvent.clear(input);
+  if (inputValue) {
+    await userEvent.type(input, inputValue);
+  }
+};
+
+/** 無効な入力を行い、エラーメッセージが表示されることを確認する */
```

Comment:
（細かいけど）
「Aを行う」は冗長な表現で、「Aする」にできます。
冗長な表現じゃないと落ち着かないということは、情報が足りてないということ。今回の場合は目的語が抜けてますね！

```suggestion
/** 無効な文字列を入力し、エラーメッセージが表示されることを確認する */
```